public/index.html

```diff
@@ -12,9 +12,10 @@
     <meta property="og:title" content="Face API" />
     <meta property="og:type" content="website" />
     <meta property="og:description" content="Face recognition API for the browser." />
-    <!-- <meta property="og:image" content="" /> -->
+    <meta property="og:image" content="/ogImage.svg" />
```

Comment:
Need png image. OG doesn't support svg format. ([link](https://indieweb.org/The-Open-Graph-protocol#Does_not_support_SVG_images))

Reply:
PTAL!